src/fpm_sources.f90

```diff
@@ -97,6 +97,9 @@ subroutine add_sources_from_dir(sources,directory,scope,with_executables,recurse
 
         dir_sources(i)%unit_scope = scope
 
+        ! TODO: Add actual link_libraries ?
```

Comment:
```suggestion
```
We can remove this comment since `link_libraries` are populated in `add_executable_sources` routine.

Reply:
Yes, it is (re)allocation-on-assignment.